.github/workflows/golangci-lint.yaml

```
@@ -54,7 +54,7 @@ jobs:
     steps:
       - uses: actions/setup-go@v5
         with:
-         go-version: "1.21"
+         go-version: "1.22.2"
          check-latest: true
```

```suggestion
         check-latest: true
```

even though we have above field still it was pulling go `1.22.1` instead of 1.22.2 which has the fix

---

@travisn also should we move to 1.22 by default in go mod?

---

> even though we have above field still it was pulling go 1.22.1 instead of 1.22.2 which has the fix

That's annoying, it would be nice if it actually pulled the latest!

> @travisn also should we move to 1.22 by default in go mod?

Let's discuss, but as a separate PR so we can get this fix merged.